### PR TITLE
Untallucas design

### PR DESCRIPTION
**Pull request by Cosmitar**
#### What does this PR do?
- Adds UI design.
- Setup of environment to process scss files.
- Adds React animations for main screens.
#### Any background context you want to provide?

do `npm install` , there're new dependency packages.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -17,20 +18,31 @@ class App extends Component {
...
         );
     }
 
     _getHomeLayout() {
         return (
-            <div className='home-wrapper'>
+            <div className='home-wrapper' key={this.props.view}>
                 <div className='home-body'>
-                    <span>Some text here</span>
+                    <svg xmlns='http://www.w3.org/2000/svg' x='0px' y='0px' width='170px' height='70px' viewBox='0 0 170 70' className='intro-logo'>
```
Comment (loverajoel) [inline src/App.js]: we should move this svg to an another component

File: src/components/results/ResultList.js [hunk 2]
```diff
@@ -2,16 +2,43 @@
...
+        let firstSong = this.props.resultList.shift();
+        let firstSongName = firstSong.name;
+        let firstArtistName = firstSong.artists[0].name;
         return (
             <div className={'results-list' + (this.props.isSearching ? 'search-mask' : '') }>
-                <span>{this.props.resultList.length}</span>
+                <div className='main-result'>
+                    <div className='track-number'>1</div>
+                    <div className='track-play-icon'>
+                        <svg x='0px' y='0px' width='40px' height='40px' viewBox='0 0 40 40'>
```
Comment (loverajoel) [inline src/components/results/ResultList.js:18]: same here

File: src/components/results/ResultList.js [hunk 3]
```diff
@@ -2,16 +2,43 @@
...
+                </button>
+
                 <ul>
                     {this.props.resultList.map(track => {
                         return (
-                            <li key={track.id}>{track.name} - ({track.artists[0].name})</li>
+                            <li key={track.id} className='single-track'>
+                                <div className='track-number'>{pos++}</div>
+                                <div className='track-play-icon'>
+                                    <svg x='0px' y='0px' width='40px' height='40px' viewBox='0 0 40 40'>
```
Comment (loverajoel) [inline src/components/results/ResultList.js:35]: same here

File: src/components/search/Search.js [hunk 4]
```diff
@@ -10,9 +10,13 @@ class Search extends Component {
                 <input
                     type='text'
                     ref='searchBox'
+                    placeholder='Just type a song name and press Enter'
                     onKeyPress={this._handleKeyPress.bind(this)}
                 />
-                <span>{this.props.searchText}</span>
+                <svg xmlns='http://www.w3.org/2000/svg' x='0px' y='0px' width='24px' height='24px' viewBox='0 0 24 24' className='search-icon'>
```
Comment (loverajoel) [inline src/components/search/Search.js:16]: same here

**Comment by loverajoel** [comment]
:shipit: after cosmetic changes

**Comment by Cosmitar** [comment]
App logo svg was moved into React Component, ohters svg will be changed in the future.

**Pull request merged**